PROCESSORS.md

```diff
@@ -20,10 +20,11 @@
 - [DeleteAzureDataLakeStorage](#deleteazuredatalakestorage)
 - [DeleteGCSObject](#deletegcsobject)
 - [DeleteS3Object](#deletes3object)
+- [ExecuteLuaScript](#executeluascript)
 - [ExecuteProcess](#executeprocess)
 - [ExecutePythonProcessor](#executepythonprocessor)
+- [ExecutePythonScript](#executepythonscript)
 - [ExecuteSQL](#executesql)
-- [ExecuteScript](#executescript)
 - [ExtractText](#extracttext)
```

Comment:
We should go back to a single processor in the docs as well.

Reply:
Good catch, fixed it in https://github.com/apache/nifi-minifi-cpp/pull/1504/commits/da748bf5a803d8599f56794e662bb4a7c9e7bf07